clang/lib/Tooling/Inclusions/Stdlib/StdSymbolMap.inc

```diff
@@ -598,7 +598,6 @@ SYMBOL(aligned_union_t, std::, <type_traits>)
 SYMBOL(alignment_of, std::, <type_traits>)
 SYMBOL(alignment_of_v, std::, <type_traits>)
 SYMBOL(all_of, std::, <algorithm>)
-SYMBOL(allocate_at_least, std::, <memory>)
```

Comment:
Was replaced with `std::allocator_traits<>::allocate_at_least` in [P2652R2](https://www.open-std.org/jtc1/sc22/wg21/docs/papers/2023/p2652r2.html)